RoundsWithFriends/RoundsWithFriends.cs

```diff
@@ -126,7 +145,15 @@ public int MaxTeams
         {
             get
             {
-                return GameModeManager.CurrentHandlerID == "Deathmatch" ? this.MaxPlayers : 2;
+                return GameModeManager.CurrentHandlerID == "Deathmatch" ? this.MaxPlayers : 4;
```

Comment:
Wait, so 1v1v1v1v2 is not allowed?

Reply:
Those values were (and are) never respected. Will be adding support for respecting them in the process of addressing some of your other comments.